src/pages/projectSettings/tabs/GeneralTab/Fields/DeactivateStepbackTasksField.tsx

```diff
@@ -50,11 +58,23 @@ const Modal: React.VFC<ModalProps> = ({ closeModal, open, projectId }) => {
       onCancel={closeModal}
       onConfirm={onConfirm}
       open={open}
-      submitDisabled={loading}
+      submitDisabled={hasError || loading}
       title="Deactivate Scheduled Stepback Tasks"
+      data-cy="deactivate-stepback-modal"
     >
-      Are you sure you would like to deactivate currently scheduled stepback
-      tasks?
+      <p>
+        Specify the build variant and task for which you want to deactivate the
```

Comment:
Would it be unreasonably complex to add https://jira.mongodb.org/browse/EVG-17951 as part of this? I think a description like this could be helpful: 
Will deactivate running stepback tasks for the specified task. To deactivate stepback more broadly, toggle stepback off for the project.

(ALternatively we could just add that second sentence to that ticket as a reminder)

Reply:
I started working on this, and it is not unreasonably complex, but I think I would prefer to open a separate PR. I added this ticket to my sprint!